gci/gitorg.py

```diff
@@ -52,7 +52,7 @@ def get_issue(url):
         return
     try:
         issue = repo.get_issue(int(issue_number))
-    except Exception as e:
+    except Exception:
```

Comment:
log the exception in the next line

Reply:
How and where should I log them?

Reply:
`e` holds the exception. Don't remove it, log it. `logger.error` in the next line does that.